tools/nl/embeddings/build_embeddings.py

```diff
@@ -180,23 +182,15 @@ def build(ctx, curated_input_dirs: List[str], local_merged_filepath: str,
   else:
     df_svs = pd.DataFrame()
 
-  # Append autogen CSVs if any.
-  autogen_dfs = []
-  for autogen_csv in sorted(glob.glob(autogen_input_filepattern)):
-    print(f'Processing autogen input file: {autogen_csv}')
-    autogen_dfs.append(pd.read_csv(autogen_csv).fillna(""))
-  if autogen_dfs:
-    df_svs = pd.concat([df_svs] + autogen_dfs)
-    df_svs = df_svs.drop_duplicates(subset=utils.DCID_COL)
```

Comment:
If we're getting rid of autogen files support, should we also delete them?

Reply:
yes, they have been deleted already...